index.html

```diff
@@ -0,0 +1,189 @@
+                        <p class="here">Explore the taste<br>of the great korean dishes of all time</p>
+                        <i class="fas fa-arrow-right"></i>
+                    </div>
+                </div>
+            </div>
+            <br><br>
+    </section>
+    <br><br>
+
+    <!-- Here is the testimonial section  -->
+    <div class="story">
+        <h4>Testimonials</h4>
+        <hr>
+    </div><br>
+    <section class="testimonials">
+        <div class="story">
+            <p>What people are saying</p>
+        </div>
+    <div class="fieldseter">
+        <fieldset>
+            <legend><img src="img/_DSC6887-Edit-Edit-Edit.jpg" alt="customers" width="90px" height="90px"></legend>
+            <h4>Sola Davies</h4>
+            <p>"Excellent service, attentive staff, and creations exceeded our expectations with a cozy atmosphere which made dining experience truly enjoyable."</p>
+            <p class="star">
+                <i class="fas fa-star"></i>
+                <i class="fas fa-star"></i>
+                <i class="fas fa-star"></i>
+                <i class="fas fa-star"></i>
+                <i class="fas fa-star"></i>
+            </p>
+        </fieldset>
+        <fieldset>
+            <legend><img src="img/_DSC8915-Edit-Edit (1).jpg" alt="customers" width="90px" height="90px"></legend>
+            <h4>Mirian Brown</h4>
+            <p>"Impeccable presentation and high quality ingredients showcased the restaurant's commitment to culinary excellence."</p>
+            <p class="star">
+                <i class="fas fa-star"></i>
+                <i class="fas fa-star"></i>
+                <i class="fas fa-star"></i>
+                <i class="fas fa-star"></i>
+                <i class="fas fa-star-half-stroke"></i>
+            </p>
+        </fieldset>
+        <fieldset>
+            <legend><img src="img/istockphoto-1448979924-170667a.jpg" alt="customers" width="90px" height="90px"></legend>
+            <h4>Slivian Daniels</h4>
+            <p>"A hidden gem! This restaurant offers a unique blend of international flavors that satisfied our adventurous palatables"</p>
+            <p class="star">
+                <i class="fas fa-star"></i>
+                <i class="fas fa-star"></i>
+                <i class="fas fa-star"></i>
+                <i class="fas fa-star"></i>
+                <i class="fas fa-star"></i>
+            </p>
+        </fieldset>
+    </div>
+    </section>
+
+    <!-- Partner Section -->
+    <div class="story">
+        <h4>Our Partners</h4>
+        <hr>
+    </div><br>
+    <section class="Partner">
+        <div class="dishes">
+            <h3 class="explore">Our Trusted<hr width="110px"></h3>
+            <h3 class="explore">Partners</h3>
+        </div><br>
+
+            <div class="filder">
+                
+                <img src="img/partner 1.png" alt="" > &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
```

Comment:
I have mentioned accessibility several times but you still do not have alt texts for your images?